Habitica Intents/IntentHandler.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  IntentHandler.swift
+//  Habitica Intents
+//
+//  Created by Christopher Coffin on 5/12/19.
+//  Copyright © 2019 HabitRPG Inc. All rights reserved.
+//
+/* Sample intents
+ Add the following under Siri intent query:
+ 
+ Add Task
+ Add get grocieries to my todo list in Habitica
+ 
+ Show Tasks in a given list
+ Show my todo list in Habitica
+ */
+
+import Intents
+
+
+class IntentHandler: INExtension, INAddTasksIntentHandling, INSearchForNotebookItemsIntentHandling {
+    
+    override func handler(for intent: INIntent) -> Any {
+        // This is the default implementation.  If you want different objects to handle different intents,
+        // you can override this and return the handler you want for that particular intent.
+        
+        return self
+    }
+
+    // handle for reading the list of task in todo
+    func handle(intent: INSearchForNotebookItemsIntent, completion: @escaping (INSearchForNotebookItemsIntentResponse) -> Void) {
+        
+        let userActivity = NSUserActivity(activityType: NSStringFromClass(INSearchForNotebookItemsIntent.self))
+        //let response = INSearchForNotebookItemsIntentResponse(code: .inProgress, userActivity: userActivity)
+        let response = INSearchForNotebookItemsIntentResponse(code: .success, userActivity: userActivity)
+        // Initialize with found message's attributes
+        response.tasks = []
+        TaskManager.shared.tasksForList(withName: "todo", oncompletion: {(taskTitles) in
```

Comment:
Is this only for todos right now because of a limitation in the siri api or just because that's the part you implemented so far?

Reply:
It's not because of a limitation in Siri, I just wanted to limit the first pass as much as possible. It should be trivial to get habits and dailies as well. I'm adding some error checking, and I'll roll the extra items into that.